coralproject/talk
# Additional Replies Increment

## What does this PR do?
Allows the number of additional replies shown on expand replies action to be configurable.  It replaces the hardcoded more replies limit: `999999` with the variable: `ADDTL_REPLIES_ON_LOAD_MORE`, that can be overridden with a new environment variable: `TALK_ADDTL_REPLIES_ON_LOAD_MORE`

## How do I test this PR?

- Add the new environment variable and give it a number.
- Find a comment with 3+ replies and click/tap on the `Show more replies` link.
- Each time this link is tapped, verify you are presented with a number of replies equal to the new environment variable value.  If the number of additional replies > # of additional replies on expand, display # additional replies.  If the number of additional replies <= # of additional replies on expand, then all remaining replies should be revealed.
